### PR TITLE
readme: doc how to run locally, also add sections

### PR DESCRIPTION
**Pull request by jeffkaufman**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,5 +25,12 @@ of bounds.
...
-to merging a PR) fails, saying they're the winner.
+## Running locally
+
+validate.py depends on some environment variables that TravisCI sets.  To run
+locally you can do:
+
+    TRAVIS_PULL_REQUEST_SHA=$(git log -1 --format='%H') \
+    TRAVIS_PULL_REQUEST=<your PR number> \
+    TRAVIS_REPO_SLUG=jeffkaufman/nomic \
+    python3 validate.py
```
[the remark this answers is not in the document]
Comment (jeffkaufman) [reply README.md]: Running validation on master (currently) just verifies that your PR has the right approvals.  If you're making a change to validation that gives interesting behavior you probably want to test out that new behavior, not what's on master.

[the remark this answers is not in the document]
Comment (jmitchell) [reply README.md]: Sometimes checking whether a PR is valid may depend on how that PR differs from `master`. For instance, while testing a WIP feature I have in mind to automatically accept changes which only add the author's handle to `players.txt`, compared to what's currently on `master`, I had to simulate potential contents of `git show master:players.txt` by manually changing `players.txt` in my local test branch.

This is an edge case, though, and I'm happy to approve as-is. More improvements can be made later if we decide it's important.

**Comment by jeffkaufman** [comment]
Merged from master.  Needs re-approval from everyone: @TheJhyde @guoruibiao @jmitchell @sligocki
@dchudz @jeffkaufman @pavellishin @tnelling

**Comment by pavellishin** [comment]
I think we should prioritize merging this in. Ping @vesche @guoruibiao
